Makefile

```diff
@@ -14,10 +14,17 @@ ifeq (${CI}, true)
 	endif
 endif
 
+SANDBOX_TESTING = sandbox-cgroup
+ifeq (${CI}, true)
+        ifneq (${TEST_SANDBOX}, true)
+                SANDBOX_TESTING =
+        endif
+endif
+
 # union for 'make test'
 UNION := functional debug-console $(DOCKER_DEPENDENCY) openshift crio docker-compose network \
 	docker-stability oci netmon kubernetes swarm vm-factory \
-	entropy ramdisk shimv2 tracing time-drift
+	entropy ramdisk shimv2 tracing time-drift $(SANDBOX_TESTING)
```

Comment:
We need to make sure to not run docker tests twice on same job as this will increase by ~30 min the execution time. I think we should only run this on one job that already runs docker tests, but with the configuration change.